strings/tips.txt

```diff
@@ -191,6 +194,7 @@ As a Terror Spider, don't forget you can move your nest if the crew is pressurin
 As a Terror Spider, try putting webs on white floor tiles, they are much harder to see there.
 As a Terror Spider, sometimes quickly pulling an enemy into a web can ensure victory in melee combat.
 You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.
+When you are restrained to alien nest and there is somebody in a nest next to you, you can unbuckle each other but not yourself.
```

Comment:
I don't think this tip should be added - this functionality is more so a bug than intentional behaviour and not something that should really be encouraged.